Dockerfile

```
@@ -1,15 +1,39 @@
-FROM python:3-slim-bullseye
+FROM python:3.13-slim-bookworm
 
-ENV HELM_DOCS_VERSION=1.14.2
+ARG HELM_DOCS_VERSION=1.14.2
 
-RUN apt update && apt install curl git -y
+# Install system dependencies and clean up in single layer
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)